legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-athena/legend-engine-xt-relationalStore-athena-execution/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/authentication/strategy/keys/AthenaUserNamePasswordAuthenticationStrategyKey.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.plan.execution.stores.relational.connection.authentication.strategy.keys;
+
+import java.util.Objects;
+
+public class AthenaUserNamePasswordAuthenticationStrategyKey implements AuthenticationStrategyKey
+{
+    private final String userNameVaultReference;
+    private final String passwordVaultReference;
+
+    public AthenaUserNamePasswordAuthenticationStrategyKey(String userNameVaultReference, String passwordVaultReference)
```

Comment:
Why we need a specialized athena auth strategy for username / password when we have a generic one that should fit here?  

https://github.com/finos/legend-engine/blob/master/legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-execution/legend-engine-xt-relationalStore-executionPlan-connection/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/authentication/strategy/UserNamePasswordAuthenticationStrategy.java